plugins/grpc-transport/src/test/java/com/github/bsideup/liiklus/transport/grpc/config/GRPCConfigurationTest.java

```diff
@@ -67,4 +73,32 @@ void shouldValidateParameters() {
         });
     }
 
+    @Test
+    void shouldConsiderTransportConfigurers() {
+        var service = ServerServiceDefinition.builder("test").build();
+
+        new ApplicationContextRunner()
+                .withInitializer((ApplicationContextInitializer) new GRPCConfiguration())
+                .withPropertyValues(
+                        "spring.profiles.active: gateway",
+                        "grpc.port: 0"
+                )
+                .withInitializer(ctx -> {
+                    var context = (GenericApplicationContext) ctx;
+                    context.registerBean(LiiklusService.class, () -> Mockito.mock(LiiklusService.class));
+                    context.registerBean(GRPCLiiklusTransportConfigurer.class, () -> builder -> builder.addService(() -> service));
+                })
+                .run(context -> {
+                    assertThat(context)
+                            .hasNotFailed()
+                            .hasSingleBean(GRPCLiiklusService.class)
+                            .hasSingleBean(GRPCLiiklusTransportConfigurer.class);
```

Comment:
both assumptions should not be `hasSingleBean`, will break if the configuration also registers configurers or more services